.agent/skills/i18n/scripts/i18n_sync.py

```
@@ -0,0 +1,163 @@
+    """Delete nested key by dot-separated key path"""
+    keys = key_path.split(".")
+    current = obj
+    for key in keys[:-1]:
+        if key not in current:
+            return
+        current = current[key]
+    if keys[-1] in current:
+        del current[keys[-1]]
+
+
+def sync_locales(dry_run: bool = True, translate: bool = False):
```

Ruff (0.15.0)

[warning] Unused function argument: `translate`

(ARG001)